local/gugcat/renderer.php

```diff
@@ -292,7 +297,7 @@ private function display_table($rows, $columns, $history=false) {
         return $html;
     }
 
-    private function context_actions($studentno, $ishidden=null, $is_aggregrade = false, $link = null, $is_overviewpage = false) {
+    private function context_actions($studentno, $ishidden=null, $is_aggregrade = false, $link = null, $is_overviewpage = false, $historyurl = null, $is_historical = false) {
```

Comment:
Use the $link parameter in passing the url, instead of adding a new one specific only to historyurl.